src/Calculator.tsx

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+
+enum Operator {
+  empty = '',
+  plus = 'plus',
+  minus = 'minus',
+  multiply = 'multiply',
+  divide = 'divide',
+}
+
+type State = {
+  prevNumber: null | number;
+  nextNumber: null | number;
+  operator: Operator;
+  result: string;
+};
+
+class Calculator extends React.Component<any, State> {
```

Comment:
```suggestion
class Calculator extends Component<any, State> {
```


react import 문에서 React 생략과 React. 떼고 개별적으로 모듈을 불러와 쓸 수 있어요.  🙂

```js
import { Component, MouseEvent } from 'react';
```